### DISCUSSION TITLE
Any reason to not consume Terminal.GUI as a package instead of via copy-posted code?

### DISCUSSION THREAD

**Discussion by tomlm** [Consolonia/Consolonia#116] [Ideas]
https://github.com/gui-cs/Terminal.Gui

**Comment by jinek** [comment]
@tomlm  The only thing I remember was I did not want to bring excess dependencies. On my mind I had an idea to discuss to extract a smaller package from GUI.cs

Overall, we surely can re-use the package, but then we need to analyze the impact: 
1) which limitations are we going to hit? For instance, does it prevent us from building mono package?
2) What would be the size increase? If we are growing from 100kb to 1mb, then lets discuss better to extract smaller package from GUI.CS
